docs/manual/manual_backend.md

```diff
@@ -588,6 +588,9 @@ Below we show how the {py:class}`~pytket.extensions.cirq.CirqStateSampleBackend`
 
 
 ```{code-cell} ipython3
+---
```

Comment:
This cell currently gives a warning because of https://github.com/CQCL/pytket-docs/issues/383, it should be enabled again after the issues is solved.